linker/shared/src/main/scala/org/scalajs/linker/backend/javascript/SourceMapWriter.scala

```diff
@@ -39,18 +40,18 @@ object SourceMapWriter {
   private final class NodePosStack {
     private var topIndex: Int = -1
     private var posStack: Array[Position] = new Array(128)
-    private var nameStack: Array[String] = new Array(128)
+    private var nameStack: Array[OriginalName] = new Array(128)
```

Comment:
Ah, ah. I did not realize this. I've reverted the first commit that turned this into an original name.

It probably does not show up in the performance profile because this happens pre-cache (almost exclusively). So in an incremental run, we only hit this code path for very little code.